andreeapetcu/assignment3/TestWordSearch.java

```diff
@@ -0,0 +1,407 @@
+import java.util.*;
+
```

Comment:
Optional:
It's great you have written tests. JUnit has become pretty much an industry standard for testing Java code. Next assignment (or this one if you have time) why don't you try uisng JUnit and see what you think. It is a very useful skill to have.